### PR TITLE
Added colour to h1

### PR DESCRIPTION
**Pull request by dex-mendeley**
Fixed the colour because I felt like it

### CODE REVIEW

**Review by hawkinsJack — APPROVED** [review]

**Review by abhatt-dot — APPROVED** [review]

**Review by tomjsillah — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -17,10 +17,14 @@ <h2>Welcome Tech Associates!</h2>
         align-items: center;
     }
 
+    h1 {
+	    color: pink;
```
Comment (dex-mendeley) [inline index.html:21]: Pink sucks, go for a different colour

File: index.html [hunk 2]
```diff
@@ -17,10 +17,14 @@ <h2>Welcome Tech Associates!</h2>
...
+
     img {
         margin-top: 3rem;
         height: 12rem;
     }
 </style>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (dex-mendeley) [inline index.html:30]: What the hell is this?

**Review by dex-mendeley — COMMENTED** [review]
Foobar

File: index.html [hunk 3]
```diff
@@ -17,10 +17,14 @@ <h2>Welcome Tech Associates!</h2>
         align-items: center;
     }
 
+    h1 {
+	    color: pink;
+    }
```
Comment (jamespen2504) [inline index.html:22]: Happy with this

**Review by jamespen2504 — APPROVED** [review]

**Review by tryfonosa — APPROVED** [review]

**Pull request merged**